src/data/matrix/notation.lean

```diff
@@ -25,6 +26,8 @@ already appears in the input.
 We reuse notation `![a, b]` for `vec_cons a (vec_cons b vec_empty)`. It is a localized notation in
 the `matrix` locale.
 
+The locale `matrix` further provides notation `⬝ᵥ` for `matrix.dot_product`.
```

Comment:
If you're going to add this new notation, can you do it in `data/matrix/basic`, and change all the existing lemmas to use the notation? It would probably make sense to do this in a standalone separate PR.

Reply:
There have already been too many commits. I will do the PR for `data/matrix/basic` after we merge this commit, would it be OK?

Reply:
That will be many trivial changes. Let's do it separately.

Reply:
> That will be many trivial changes. Let's do it separately.

Yes, that's exactly what I'm suggesting; do it separately, but also do it *first* (ie, pause this PR until its done), as it means the decision to add the notation can be reviewed in isolation.

Reply:
https://github.com/leanprover-community/mathlib/pull/11289

Reply:
Thanks! This line can now be removed, right?